packages/cli/src/request.ts

```diff
@@ -3,76 +3,26 @@ import ora from 'ora';
 
 const spinner = ora();
 
-interface RequestParams {
-  [key: string]: any;
-}
+export type RequestParams = Record<string, any>;
 
-export interface ResponseParams {
-  [key: string]: any;
-}
+export type ResponseParams = Record<string, any>;
 
-export async function get(host: string, params?: RequestParams): Promise<ResponseParams> {
+const createGeneralRqeuest = (agent: Function) => async (...args: any[]) => {
```

Comment:
Typo `Rqeuest`,  this was introduced by my example, sorry :(

Reply:
Fix it:)